packages/components/src/components/form-progress/form-progress.jsx

```diff
@@ -3,90 +3,79 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import Text from '../text';
 
-class FormProgress extends React.PureComponent {
-    constructor(props) {
-        super(props);
-        this.el_completed_bar = React.createRef();
-    }
+const FormProgress = ({ steps, current_step }) => {
+    React.useEffect(() => {
+        animateCompleteBar();
+    });
 
-    componentDidMount() {
-        this.animateCompleteBar();
-    }
+    const el_completed_bar = React.createRef();
```

Comment:
We don't need to have `createRef()` in functional components anymore as it is already taken care of, instead we need to have a `useRef` hook.

you can have a look here for more details: https://stackoverflow.com/questions/54620698/whats-the-difference-between-useref-and-createref

Reply:
got it, thx!